packages/blaze-hot/hot.js

```diff
@@ -38,31 +38,31 @@ function patchTemplate(Template) {
     }
 
     return oldOnDestroyed.call(this, cb);
-  }
+  };
 
   const oldHelpers = Template.prototype.helpers;
   Template.prototype.helpers = function (dict) {
     if (typeof dict === 'object') {
-      for (var k in dict) {
+      for (let k in dict) {
```

Comment:
please add a TODO note, that we should change this in the future to a solution that won't use for..in

Reply:
Probably can do this in this PR, I have fixed that in other PRs.